packages/plugin-compat/sources/extensions.ts

```diff
@@ -162,4 +162,10 @@ export const packageExtensions: Array<[string, any]> = [
       'react-dom': `>= 16.3.0`,
     },
   }],
+  // https://github.com/apollographql/apollo-server/pull/4682
+  [`apollo-upload-client@<14`, {
+    peerDependencies: {
+      graphql: `14 - 15`,
```

Comment:
Shouldn't that be `^0.11.3 || ^0.12.3 || ^0.13.0 || ^14.0.0 || ^15.0.0`?

Reply:
It's declared like that upstream https://github.com/jaydenseric/apollo-upload-client/blob/6dec8edc2bb1c97514e1577b84530922a40608bc/package.json#L47-L49, though it seems invalid :thinking: